docs/understanding-the-flutterflow-ui/Projects/how-to-collaborate-on-projects.md

```diff
@@ -0,0 +1,163 @@
+---
+slug: how-to-collaborate-on-projects
+title: How to Collaborate on Projects
+tags: []
+toc_max_heading_level: 5
+sidebar_position: 4
+
+---
+FlutterFlow allows for multiple ways to collaborate on a project as well as the ability to manage the collaboration through permissions.
+
+## Sharing a Project with a User
+
+You can also invite users to your project who are not part of your organization. For instance, you might want to share your work with clients, stakeholders, or team members of the client.
+
+When you invite users to your project, you have the flexibility to assign them different roles depending on their level of involvement. For example, you can grant editing privileges to collaborators who are actively working on the project while assigning read-only access who need to view the progress. Alternatively, you can also make them the owner of the project, giving them full control.
+:::info
+- When you add a user, the default role assigned is the *Editor* role.
+- Users with read-only access will only be able to access that specific project and won't be able to access any shared *Teams* libraries (e.g., custom code, design system).
+- Read-only users added to *Teams*/*Organization* will not count against your *Teams* seat count.
+- You must verify your email before inviting users.
+  :::
+
+### How to Invite Users
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/9XkpLuC3tpiFFapWi7ao?embed&show_copy_link=true"
+        title="Sharing a Project with a User"
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light',
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+
+1. Navigate to **Settings & Integrations** (from the left *Navigation Menu*) **> Project Setup > Collaboration > Invite Users**.
```

Comment:
Steps should be part of the Arcade itself. The entire idea is to remove too much of steps as text! But if you want to explain something further, definitely mention it as text after your Arcade. But most "click this and click that" and one liner explanation can be part of Arcade callouts itself. 

Here's a retool example Leigha shared in Pinkesh's PR - https://docs.retool.com/apps/web?docid=landing
You can check the quickstart arcade or authentication Arcades as reference.

Reply:
done